nnef/src/ops/resource.rs

```diff
@@ -0,0 +1,23 @@
+use crate::internal::*;
+
+pub fn register(registry: &mut Registry) {
+    registry.register_primitive(
+        "tract_resource_get",
+        &[
+            TypeName::String.named("label"),
+            TypeName::String.named("key"),
+        ],
+        resource_get,
+    );
+}
+
+fn resource_get(
+    builder: &mut ModelBuilder,
+    invocation: &ResolvedInvocation,
+) -> TractResult<Value> {
+    let label: String = invocation.named_arg_as(builder, "label")?;
+    let key: String = invocation.named_arg_as(builder, "key")?;
+    let resource = builder.proto_model.resources.get(&label)
+        .with_context(|| anyhow!("No resource found for label {:?} in the model", label))?;
+    resource.get(&key)
+}
```

Comment:
Can we hand-craft a couple of usage examples ? as test-cases maybe ? either inline (but It may be complicated) or as integration tests (to make it easier to have external files)